lib/fluent/plugin/out_dynatrace.rb

```diff
@@ -64,7 +64,12 @@ def configure(conf)
         compat_parameters_convert(conf, :inject)
         super
 
+        raise Fluent::ConfigError, 'active_gate_url has not been set' if @active_gate_url == ''
+
         @uri = URI.parse(@active_gate_url)
+        raise Fluent::ConfigError, 'active_gate_url scheme must be http or https' unless
```

Comment:
This would crash if I set it up to be `127.0.0.1` or some other IP right? I guess its easily fixed by just adding the `http`/`https` there anyway.

Reply:
Yes, it is now required to add the scheme to the `active_gate_url`. Is omitting the scheme something that we want to allow? :thinking:

Reply:
That's probably not necessary. Maybe we could add a happy path test for IPv6, but I think it should work (e.g. `http://[::1]/`)

Reply:
With throwing one will quickly find out now that they missed adding the scheme, so I think we don't need to guess which of the schemes it should be.

Reply:
Test added in [4fc8661](https://github.com/dynatrace-oss/fluent-plugin-dynatrace/pull/34/commits/4fc8661e67b08bf90759df9f8e3d97ade71dff01) :slightly_smiling_face: